src/site/topic18.rst

```diff
@@ -6,6 +6,34 @@ Topic #18 --- Memory & The Call Stack
 Memory Allocation
 =================
 
+* Memory is broken down into two broad sections
+
+1. The Stack
```

Comment:
May be worth mentioning that primitive types also live on the stack (or rather, they are not references so share memory space with their parent block).

Reply:
I'll hit that in more detail when I get to the STACK section.